React-Native/TodoList/src/app/screens/Main/components/Footer/styles.js

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: 'lightblue',
+    padding: 15
+  },
+  removeLabel: {
+    textAlign: 'center',
+    color: 'red'
```

Comment:
add a constant file for colors, called colors.js.. there you can define this one and all you have